extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -2126,64 +2231,47 @@ describes.realWin('Events', {amp: 1}, (env) => {
     });
 
     describe('should wait on correct readyPromise', () => {
-      const selector = '.target';
-
       it('with waitFor default value', () => {
         // Default case: selector is not specified
-        expect(tracker.getReadyPromise(undefined, undefined)).to.be.null;
+        expect(tracker.getReadyPromise(undefined)).to.be.null;
         // Default case: waitFor is not specified, no AMP element selected
         iniLoadTrackerMock
           .expects('getRootSignal')
           .returns(Promise.resolve())
           .once();
-        const waitForTracker1 = tracker.getReadyPromise(undefined, ':root');
+        const waitForTracker1 = tracker.getReadyPromise('ini-load');
         return waitForTracker1.then(() => {
           iniLoadTrackerMock
             .expects('getElementSignal')
             .withExactArgs('ini-load', target)
             .returns(Promise.resolve())
             .once();
           // Default case: waitFor is not specified, AMP element selected
-          const promise2 = tracker.getReadyPromise(undefined, selector, target);
+          const promise2 = tracker.getReadyPromise('ini-load', target);
           target.signals().signal('ini-load');
           return promise2;
         });
       });
 
       it('with waitFor NONE', () => {
-        expect(tracker.getReadyPromise('none', undefined, undefined)).to.be
-          .null;
-        expect(tracker.getReadyPromise('none', ':root', undefined)).to.be.null;
-        expect(tracker.getReadyPromise('none', selector, target)).to.be.null;
```

Comment:
we can leave this one?

Reply:
We've changed `getReadyPromise()` to only take a `waitForSpec` that gets calculated inside `addTrigger()`. See `readyPromiseWaitForSpec`.

All three of these expectations translate to         `expect(tracker.getReadyPromise('none')).to.be.null;` in this new version.

Reply:
True. Sounds good.